gf-webapp-admin/src/component/filter/Filters.tsx

```diff
@@ -0,0 +1,57 @@
+import {Filter} from "src/component/filter/Filter";
+import {URL_QUERY_KEYS} from "src/component/filter/FilterInput";
+import {useFilterRooms} from "src/component/filter/FilterContext";
+import {getCurrentDate, getNextDate} from "src/utils/getDate";
+import styles from "src/component/filter/Filters.module.scss";
+
+const MIN_ADULTS_VALUE = 0;
+const MIN_CHILDREN_VALUE = 0;
+export const defaultAdultsValue: number = MIN_ADULTS_VALUE;
+export const defaultChildrenValue: number = MIN_CHILDREN_VALUE;
+export const defaultSearchValue = "";
+export const MIN_DATE_IN = getCurrentDate;
+export const MIN_DATE_OUT = getNextDate;
+
+export const Filters = () => {
```

Comment:
It would be nice to add a filter reset button to make it easy to see the full list of rooms again

Reply:
I think an administrator should be able to view all rooms, even if they are booked. Now, as far as I understand, rooms booked on default dates will not be displayed

Reply:
Add 2 sections on main page. One with all rooms and second with filtered rooms.